{
	"schema_version": "1.6.0",
	"id": "DSA-3544-1",
	"modified": "2023-10-16T00:12:06.526065Z",
	"published": "2016-04-07T00:00:00Z",
	"related": [
		"CVE-2016-2512",
		"CVE-2016-2513"
	],
	"summary": "python-django - security update",
	"details": "\nSeveral vulnerabilities were discovered in Django, a high-level Python\nweb development framework. The Common Vulnerabilities and Exposures\nproject identifies the following problems:\n\n\n* [CVE-2016-2512](https://security-tracker.debian.org/tracker/CVE-2016-2512)\nMark Striemer discovered that some user-supplied redirect URLs\n containing basic authentication credentials are incorrectly handled,\n potentially allowing a remote attacker to perform a malicious\n redirect or a cross-site scripting attack.\n* [CVE-2016-2513](https://security-tracker.debian.org/tracker/CVE-2016-2513)\nSjoerd Job Postmus discovered that Django allows user enumeration\n through timing difference on password hasher work factor upgrades.\n\n\nFor the oldstable distribution (wheezy), these problems have been fixed\nin version 1.4.5-1+deb7u16.\n\n\nFor the stable distribution (jessie), these problems have been fixed in\nversion 1.7.7-1+deb8u4.\n\n\nFor the testing distribution (stretch), these problems have been fixed\nin version 1.9.4-1.\n\n\nFor the unstable distribution (sid), these problems have been fixed in\nversion 1.9.4-1.\n\n\nWe recommend that you upgrade your python-django packages.\n\n\n",
	"affected": [
		{
			"package": {
				"ecosystem": "Debian:7",
				"name": "python-django",
				"purl": "pkg:deb/debian/python-django?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1.4.5-1+deb7u16"
						}
					]
				}
			],
			"versions": [
				"1.4.5-1",
				"1.4.5-1+deb7u1",
				"1.4.5-1+deb7u10",
				"1.4.5-1+deb7u11",
				"1.4.5-1+deb7u11~bpo60+1",
				"1.4.5-1+deb7u12",
				"1.4.5-1+deb7u13",
				"1.4.5-1+deb7u13~bpo60+1",
				"1.4.5-1+deb7u14",
				"1.4.5-1+deb7u14~bpo60+1",
				"1.4.5-1+deb7u15",
				"1.4.5-1+deb7u2",
				"1.4.5-1+deb7u3",
				"1.4.5-1+deb7u4",
				"1.4.5-1+deb7u4~bpo60+1",
				"1.4.5-1+deb7u5",
				"1.4.5-1+deb7u6",
				"1.4.5-1+deb7u7",
				"1.4.5-1+deb7u7~bpo60+1",
				"1.4.5-1+deb7u8",
				"1.4.5-1+deb7u8~bpo60+1",
				"1.4.5-1+deb7u9"
			],
			"database_specific": {
				"source": "https://storage.googleapis.com/debian-osv/dsa-osv/DSA-3544-1.json"
			}
		},
		{
			"package": {
				"ecosystem": "Debian:8",
				"name": "python-django",
				"purl": "pkg:deb/debian/python-django?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1.7.7-1+deb8u4"
						}
					]
				}
			],
			"versions": [
				"1.7.7-1",
				"1.7.7-1+deb8u1",
				"1.7.7-1+deb8u2",
				"1.7.7-1+deb8u3"
			],
			"database_specific": {
				"source": "https://storage.googleapis.com/debian-osv/dsa-osv/DSA-3544-1.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://www.debian.org/security/2016/dsa-3544"
		}
	]
}
